### ISSUE TITLE
The "protobuf-lite" is incompatible with other libraries using "protobuf-javalite"

### ISSUE THREAD

**Issue by dasirra** [petri-lipponen-movesense/mdsflutter#4]
This library is using `com.google.protobuf:protobuf-lite:3.0.1`, which generates a duplicated class error when the library is used with other libraries depending on `com.google.protobuf:protobuf-javalite`, like "`cloud-firebase`" flutter plugin, a very common library in Android development.

To solve this issue, maybe an upgrade can be made in this repository to "`javalite`" protobuf serialization.

Possible solution here: https://stackoverflow.com/a/65585103/2931960
```gradle
dependencies {
    implementation  "com.google.protobuf:protobuf-javalite:3.10.0"
}

protobuf {
    protoc {
        artifact = "com.google.protobuf:protoc:3.10.0"
    }

    generateProtoTasks {
        all().each { task ->
            task.builtins {
                java {
                    option 'lite'
                }
            }
        }
    }
}
```

**Comment by petri-lipponen-movesense** [comment]
I'm unfamiliar with the protobuf stuff, so could you explain what possible issues come with the change? Also which one is the more widely used at the moment?

Also I found a post: https://github.com/protocolbuffers/protobuf/issues/8104, which talks about excluding transitive dependencies:

https://docs.gradle.org/current/userguide/dependency_downgrade_and_exclude.html#sec:excluding-transitive-deps

Which recommends a third option. Are you familiar with it?

**Comment by petri-lipponen-movesense** [comment]
I've published 2.0.0-beta2 which has this issue hopefully fixed. Please try it out.
